source/voice_control/create_wake_word.markdown

```diff
@@ -0,0 +1,103 @@
+---
+title: "Create your own wake word"
+---
+
+You can now create your own wake word. The procedure below will take you into a Python environment to train a model. The model is trained on synthetic voice clips that are generated by the local neural text-to-speech system [Piper](https://github.com/rhasspy/piper). Want to now more about how this all works? Check out the [openWakeWord](https://github.com/dscripka/openWakeWord) project by David Scripka.
+
+Depending on the word, training a model on your own wake word may take a few iterations and a bit of tweaking. This guide will take you through the process step by step.
+
+## Prerequisites
+
+- latest version of Home Assistant
+- [M5Stack ATOM Echo Development Kit](https://shop.m5stack.com/products/atom-echo-smart-speaker-dev-kit?ref=NabuCasa)
+- successfully completed the [$13 voice remote for Home Assistant](/voice_control/thirteen-usd-voice-remote/) tutorial
+
+## To create your own wake word
+
+1. Think of a wake word.
+   - A word or short phrase (3-4 syllables) that is not commonly used so that it does not trigger Home Assistant by mistake.
+   - Currently, only wake words in English are supported.
+2. Open the [wake word training environment](https://colab.research.google.com/drive/1q1oe2zOyZp7UsB3jJiQ1IFn8z5YfjwEb?usp=sharing#scrollTo=1cbqBebHXjFD).
+3. In section 1, enter your wake word in the **target_word** field.
+![Enter wake word in target field](/images/assist/wake_word_enter_target_word.png)
+4. In the code section next to the **target_word**, select the play button.
```

Comment:
Should we mention that the first time it can take 30 seconds?
```suggestion
4. In the code section next to the **target_word**, select the play button. The first time this can take up to 30 seconds.
```